mkdocs.yml

```diff
@@ -7,7 +7,7 @@ nav:
       - Security: security-levels.md
       - Data sources:
           - data-sources/index.md
-          - Overview: data-sources/intro.md 
+          - Overview: data-sources/intro.md
```

Comment:
note: These whitespace changes are now picked up by pre-commit, so we have to fix them to placate it.